samples/BotwinSample/Startup.cs

```diff
@@ -7,22 +7,14 @@ namespace Botwin.Samples
     using Microsoft.AspNetCore.Http;
     using Microsoft.Extensions.Configuration;
     using Microsoft.Extensions.DependencyInjection;
-    using Microsoft.Extensions.Logging;
 
     public class Startup
     {
-        private readonly ILoggerFactory loggerFactory;
-
-        public Startup(ILoggerFactory loggerFactory)
-        {
-            this.loggerFactory = loggerFactory;
-        }
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<IActorProvider, ActorProvider>();
             
-            //If you want to log what Botwin finds you need to pass in a ILoggerFactory as ASP.Net Core does not offer a clean way to log extensions to IServiceCollection
-            services.AddBotwin(this.loggerFactory);
+            services.AddBotwin(enableDiagnostics: true);
```

Comment:
I would make this a flag on an Options type. It should look like this:

```C#
services.AddBotwin(options =>
{
    options.EnableDiagnostics = true;
});
```

Plus it sets up Botwin for adding future options.

Reply:
Yeah, I wanted to do that too, but then I would have to call the action myself once within the `AddBotwin` to figure out whether I need to collect the types or not. And I couldn’t take the options from other sources then either.

So this would only work properly if the type collection would always be enabled and the flag just controls whether to do the logging within the `UseBotwin()` call.